GREEDY_WEB/ClientApp/components/Authorization.tsx

```diff
@@ -0,0 +1,34 @@
+﻿import * as React from 'react';
+import Cookies from 'universal-cookie';
+import { UserLogin } from './UserLogin';
+import { UserRegistration } from './UserRegistration';
+import { Switch, Route } from 'react-router-dom';
+import { ForgotPassword } from './ForgotPassword';
+
+export class Authorization extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
```

Comment:
Because this class takes in arguments from another class (the arguments are called props) and I believe in react if you want to reference those arguments correctly they need to be passed to the base react class (the one this extends from). And that means you have a constructor that calls its father constructor

Reply:
okay, thanks :)